pelican/settings.py

```diff
@@ -101,7 +101,10 @@ def load_source(name, path):
     'PELICAN_CLASS': 'pelican.Pelican',
     'DEFAULT_DATE_FORMAT': '%a %d %B %Y',
     'DATE_FORMATS': {},
-    'MD_EXTENSIONS': ['codehilite(css_class=highlight)', 'extra'],
+    'MD_EXTENSIONS': {
+        'markdown.extensions.codehilite': {'css_class': 'highlight'},
+        'markdown.extensions.extra': {},
+    },
```

Comment:
And since the dict is _updated_, does inclusion here obsolete the test in MarkdownReader?